lib/openstudio/workflow/run.rb

```diff
@@ -102,7 +102,7 @@ def initialize(osw_path, user_options = {})
         @final_message = ''
         @current_state = nil
         @options = {}
-        
+
```

Comment:
(My editor (Vim) is configured to strip trailing whitespace)

Reply:
In case you missed it, you can ignore whitespace change in github diffs by appending `?w=1` to the url: https://github.com/NREL/OpenStudio-workflow-gem/pull/92/files?w=1